rc/.eslintrc.json

```diff
@@ -247,14 +267,23 @@
         ],
         "new-parens": 2,
         "newline-after-var": 0,
-        "object-curly-spacing": [0, "never"],
+        "object-curly-spacing": [
+            2,
+            "never"
```

Comment:
As for array-spacing. I would say no; arrays and functions are the same, they are N things.

Where as objects are key values seperated by new lines. When you put them on one line you seperate them by spaces instead.